config/samples/ccruntime.yaml

```diff
@@ -7,5 +7,118 @@ spec:
   # Add fields here
   runtimeName: kata
   config:
+    cleanupNodeSelector: 
+      - matchLabels:
+        - name: "katacontainers.io/kata-runtime"
+          value: "cleanup"
     installType: bundle
-    payloadImage: quay.io/confidential-containers/runtime-payload:v0
+    payloadImage: quay.io/kata-containers/kata-deploy:stable
```

Comment:
Just a note for our future selves that deserves documentation.  Right now this image does **not** have the CC bits in.
So, it'll work for our normal kata with the patched containerd, but it's not yet material for using the CC work that has been in development.